src/main/java/jp/co/soramitsu/sora/crypto/service/impl/JSONCanonizerWithOneCoding.java

```diff
@@ -0,0 +1,19 @@
+package jp.co.soramitsu.sora.crypto.service.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import jp.co.soramitsu.jackson.OneCodeMapper;
+import jp.co.soramitsu.sora.crypto.service.JSONCanonizer;
+import lombok.NoArgsConstructor;
+
+
+@NoArgsConstructor
+public class JSONCanonizerWithOneCoding implements JSONCanonizer {
+
+  private ObjectMapper mapper = new OneCodeMapper();
```

Comment:
do we need to create an instance of ObjectMapper all the time? Maybe it is better to use a singleton?

Reply:
it is better to make the field `final`

Reply:
Singletons are evil. This one is not just ObjectMapper, but 1coding implementation (https://github.com/soramitsu/1code-java)